src/server/styling.ts

```diff
@@ -306,6 +306,21 @@ export const baseTemplate = (
     .reason-badge.previous_improved { background: #f3e5f5; color: #7b1fa2; }
     .reason-badge.new_approach { background: #e8f5e8; color: #2e7d32; }
     .count { color: #666; font-weight: normal; }
+    .button {
```

Comment:
Let's not use buttons for now. Keep everything text based. So just clickable link.